NEWS.md

```diff
@@ -1,3 +1,9 @@
+# admiraltemplate 0.0.3
+
+# Updates to site
+
+ - Package Extension guidance on in the front page was updated and moved to [this location](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html) in `{admiraldev}`, and a link was added to point to the new location (#61).
```

Comment:
we usually use the devel version of the site as default for admiraldev - so use this in the link. as it was important we always see the very latest developer guidance